.github/workflows/automatic-update.yml

```diff
@@ -43,3 +44,9 @@ jobs:
 
             **Notes for reviewer:**
             This is an auto-generated PR which ran `npx -y @grafana/create-plugin@latest update`, updates plugin.json files, grafana-plugin-sdk-go and docker-compose files for each plugin example
+
+      - name: Auto approve PR
```

Comment:
Should this live in the same workflow or in a workflow that runs on `pull_request`? 🤔

Reply:
I am not sure, it is necessary. Is it?

Reply:
I don't think it's necessary, but might be a little tidier having an action that runs with the `pull_request` event.